File: php/Aksesoris.php
<?php

require_once('Baju.php'); 

class Aksesoris extends Baju {
    // attributes
    private $jenis = "";
    private $bahan = "";
    private $warna = "";

    // constructor
    public function __construct($jenis = "", $bahan = "", $warna = "") {
        parent::__construct();
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // getter dan setter
    public function get_jenis() {
        return $this->jenis;
    }

    public function set_jenis($jenis) {
        $this->jenis = $jenis;
    }

    public function get_bahan() {
        return $this->bahan;
    }

    public function set_bahan($bahan) {
        $this->bahan = $bahan;
    }

    public function get_warna() {
        return $this->warna;
    }

    public function set_warna($warna) {
        $this->warna = $warna;
    }
}

?>

File: java/Aksesoris.java
public class Aksesoris extends Baju {
    private String jenis, bahan, warna;

    public Aksesoris() {
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public Aksesoris(String jenis, String bahan, String warna) {
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String getJenis() {
        return jenis;
    }

    public void setJenis(String jenis) {
        this.jenis = jenis;
    }

    public String getBahan() {
        return bahan;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public String getWarna() {
        return warna;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }
}